cmd/cosign/cli/options/verify.go

```diff
@@ -43,6 +44,9 @@ func (o *CommonVerifyOptions) AddFlags(cmd *cobra.Command) {
 		"ignore transparency log verification, to be used when an artifact signature has not been uploaded to the transparency log. Artifacts "+
 			"cannot be publicly verified when not included in a log")
 
+	cmd.Flags().BoolVar(&o.PrivateInfrastructure, "private-infrastructure", false,
+		"an alias for --insecure-ignore-tlog")
```

Comment:
Given we don’t want to refer to this as insecure, can you add a description instead, something like “skip transparency log verification when verifying artifacts in a privately deployed infrastructure” or something like that

Reply:
That's much better! I pushed an update with this instead